packages/manager/modules/pci/src/index.scss

```diff
@@ -4,4 +4,12 @@
   @import '~bootstrap4/scss/_mixins';
   @import '~bootstrap4/scss/_utilities';
   @import '~bootstrap4/scss/_grid';
+
+  .minh-100 {
```

Comment:
Do you still need this code with the fixs that have been applied on the branch `develop`?

Reply:
Yes I still need it because of white backgrounds that stop at 100% of the height. When scrolling the bg-white is cutted. Adding min-height 100% fix that